content/integrations/statsd.md

```diff
@@ -12,7 +12,9 @@ Enable the Datadog StatsD integration to:
   * Monitor uptime and availability of a StatsD server
   * Collecting metrics from a StatsD server's admin interface
 
-**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. This integration is intended for monitoring external StatsD servers, and is not needed to send metrics to Datadog using the StatsD protocol.**
+**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. If you want to send metrics from a statsd server, just point it to the forwarder listening on port 17123.**
```

Comment:
Re: Ilan's comment, looking at the Agent config file, I think Matt to advise forwarding StatsD-collected metrics to the Agent Forwarder (port 17123), not to DogStatsD.

But if this is right thing to recommend, it's confusing to first state that DogStatsD serves as a StatsD forwarder, and then to advise forwarding to the Agent Forwarder process.

In any case, we shouldn't provide instruction on this here. We should document it elsewhere (one of the Agent pages) if it isn't already and link to it from here.